### ISSUE TITLE
AE2 performance severely degraded in 2.2.7

### ISSUE THREAD

**Issue by 10935336** [GTNewHorizons/GT-New-Horizons-Modpack#12103]
### Your GTNH Discord Username

10935336#1951

### Your Pack Version

2.2.7

### Your Server

self host server 

### Type of Server

Vanilla Forge

### Your Expectation

performance remains normal.

### The Reality

I upgraded my server from 2.1.2.1 to 2.2.7, and I'm having serious performance issues.
Going through Spark and VisualVM I noticed that it might be an AE2 issue.
Then I replaced the appliedenergistics2-1.7.10-rv3-beta-122-GTNH.jar of 2.2.7 with appliedenergistics2-1.7.10-rv3-beta-131-GTNH-pre.jar The situation has improved but AE2 is still taking up a lot of ticktime.

VisualVM nps:
github does not allow uploading nps files so this is a compressed package.
[nps.zip](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/10303369/nps.zip)

spark report when using AE2-rv3-beta-122-GTNH:

- https://spark.lucko.me/63Esjeo5zr
- https://spark.lucko.me/8FMeQkRJdi
- https://spark.lucko.me/xrzwY08eJp

### Your Proposal

Check out the AE2 code.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [ ] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by Glease** [comment]
Duplicate of #11991

**Comment by 10935336** [comment]
I don't think it's Duplicate, I tried appliedenergistics2-1.7.10-rv3-beta-131-GTNH-pre.jar

**Comment by Glease** [comment]
What if you use 130 instead? Just trying to narrow the culprit.

**Comment by 10935336** [comment]
> What if you use 130 instead? Just trying to narrow the culprit.

130 looks worse
[nps-AE2-130.zip](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/10303529/nps-AE2-130.zip)

**Comment by Miisi** [comment]
been having the same problem, didn't notice this issue already existed so I'm posting what I already had written in the other one:


requesting certain items inexplicably drops my world's TPS to 0-1. lag persists for a while until it comes back to normal, and then if I try to request the same item then there is no lag. it's been quite hard to reproduce as I can't find anything that could be causing it in world. it is basically unplayable as there's no explanation as to when a huge tps drop is coming

![image](https://user-images.githubusercontent.com/46682682/210047832-75ce7e15-556d-41bb-ae94-fc1261a8812f.png)

(after requesting 4 dimensional bridges)

tried profiling with opis, absolutely no information in there (in fact it was not even measuring my dimension for some reason?), so I had to use visualvm (this was done with ae2 129)

![image](https://user-images.githubusercontent.com/46682682/210047843-7b700587-b3da-493f-864d-b73e62692d2d.png)


it's not every item that lags upon request, for example:

- really ultimate battery does not lag
- dimensional bridges (and anything that uses them) DO lag
- thousands of iron blocks do not lag

only changes to my crafting infrastructure before the lag started happening were a couple of patterns related to fluid solidifying

my AE2 versions were a bit outdated, so I updated every AE2 related mod (extra cells, ae2fc, not enough energistics, etc) to no difference. manii (on Discord) was also having a similar issue on Zeta where requesting a bio mainframe would bring the server to its knees

have tried updating and downgrading to multiple AE2 versions and removing every pattern with substitutions enabled, same tps drops happens (on versions 122, 129 and 132)

**Comment by POPlol333** [comment]
> > What if you use 130 instead? Just trying to narrow the culprit.
> 
> 130 looks worse. [nps-AE2-130.zip](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/10303529/nps-AE2-130.zip)

I know we aren't great exemple since with @GDCloudstrike we have a really weird mod folder but here we have AE2 130 and we aren't witnessing any issue at all

**Comment by GDCloudstrike** [comment]
>I know we aren't great exemple since with @GDCloudstrike we have a really weird mod folder but here we have AE2 130 and we aren't witnessing any issue at all

We do have this issue actually, but only with uv microwave energy transmitters and all the things that use them (dimensional bridges, stargate parts), just like what Misi experienced.

**Comment by Miisi** [comment]
yup, can confirm that trying to craft UV microwave energy transmitters brings tps down
kind of weird considering I had already been crafting them just fine without issues, maybe having prestocked lapotronic energy clusters for the energy module is somehow related?

**Comment by paulzzh** [comment]
can confirm in 2.2.8
trying to craft 120 dimensional bridges or 120 UV microwave energy transmitters
TPS drops to 0.9
[20230201-064113.zip](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/10551725/20230201-064113.zip)
